includes/admin/reports/class-gateways-reports-table.php

```diff
@@ -215,7 +215,7 @@ public function reports_data() {
 				'complete_sales'  => $complete_count,
 				'pending_sales'   => $pending_count,
 				'total_sales'     => $complete_count + $pending_count,
-				'total_donations' => give_currency_filter( give_format_amount( $stats->get_earnings( 0, 0, 0, $gateway_id ), array( 'sanitize' => false ) ) ),
+				'total_donations' => give_currency_filter( give_format_amount( $stats->get_earnings( 0, strtotime('04/13/2016' ), current_time('timestamp' ), $gateway_id ), array( 'sanitize' => false ) ) ),
```

Comment:
@DevinWalker date updated:
For Ref: https://github.com/WordImpress/Give/commit/73e6520a77cf0e0d3a9c297880465afa70a71ca2